libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -91,15 +91,31 @@ func getCpuUsageBreakdown(path string) (uint64, uint64, error) {
 	if err != nil {
 		return 0, 0, err
 	}
-	// TODO: use strings.SplitN instead.
-	fields := strings.Fields(data)
-	if len(fields) < 4 || fields[0] != userField || fields[2] != systemField {
+
+	lines := strings.SplitN(data, "\n", 2)
```

Comment:
In this case you can actually use strings.Cut (which was not available when this TODO was written).

Reply:
I added a commit that uses `strings.Cut` instead of `strings.SplitN` and `strings.Fields`.